### ISSUE TITLE
[Windows] “Crash reporting” screen not displayed when triggered from Dev options

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#6753]
**Important details:** 

* not reproducing on macOS/Android;

* not reproducing on 2.14 version;

**Affected versions:**

* 2.15.0 (2.202304210155)

**Tested Platforms:**

* Windows 10/11;

**Prerequisites:** 

* Have a Fx account with a subscription;

* Have Mozilla VPN app installed and opened;

* Be signed in - Stage server;

**Steps to reproduce:**

1. Go to Settings - Get help - Developer Options and press the “Test Crash Reporter” button 6 times;
1. Reopen the app;
1. Observe the screen;

**Expected result:**

* “Crash reporting” screen is displayed;

**Actual result:**

* “Crash reporting” screen is NOT displayed;

**Notes:**

* See the attached video and logs;

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-4670)
┆Reporter: Bianca Hidecuti

**Comment by data-sync-user** [comment]
➤ Naomi Kirby commented:

I think this is a duplicate of https://mozilla-hub.atlassian.net/browse/VPN-4409 ( https://mozilla-hub.atlassian.net/browse/VPN-4409|smart-link ) and we just need to uplift the fix to 2.15.

**Comment by data-sync-user** [comment]
➤ Beatriz Rizental Machado commented:

This is a duplicate. I am waiting on confirmation on whether or not to uplift it to the 2.15 branch. If we are not uplifiting it we can close this as WONTFIX.

**Comment by data-sync-user** [comment]
➤ Beatriz Rizental Machado commented:

Closing this as a duplicate. Since this a development only feature, we have decided it doesn't need to be in 2.15.

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

QA thinks that this should be included in 2.15, because:

  * while encountering crashes on Windows using Mozilla VPN 2.15, we did not see the Crash report screen - this is what a user would experience;
  * the ticket for witch this was considered a duplicate - https://mozilla-hub.atlassian.net/browse/VPN-4409 ( https://mozilla-hub.atlassian.net/browse/VPN-4409|smart-link ) - contained the fix for both testing feature and real flow crash feature;
  * because we did not see the crash screen, we could not click the “Send report“ button, hence a Sentry ticket was not created;
  * previous experience showed us that it is safer to have the fix not skipping a release version;

Reopening this ticket.

Beatriz Rizental Machado Naomi Kirby Matt Lichtenstein

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

Verified this on Mozilla VPN 2.15.0 (2.202305092244), while using Windows 10/11 and the “Crash reporting” screen cannot be triggered, because after pressing the “Test Crash Reporter” button 5 times, the 6th press (click) does nothing - the button shows that it should be press one more time - attaching video and logs.

Reopening this ticket.

!4670.mp4|width=496,height=964!

[^4670.txt]

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

Verified this as fixed on Mozilla VPN 2.15.0 (2.202305110046), using Windows 10/11 and the “Crash reporter” screen is now displayed when triggered via the button from Dev options.

Attaching postfix video.

!bandicam 2023-05-11 08-43-56-058.mp4|width=496,height=956!
